Library/Homebrew/cask/test/cask/artifact/alt_target_test.rb

```diff
@@ -2,7 +2,7 @@
 
 describe Hbc::Artifact::App do
   describe "activate to alternate target" do
-    let(:cask) { Hbc.load("with-alt-target") }
+    let(:cask) { Hbc::CaskLoader.load_from_file(TEST_FIXTURE_DIR.join("cask", "Casks", "with-alt-target.rb")) }
```

Comment:
`Hbc::CaskLoader.load_from_file("#{TEST_FIXTURE_DIR}/cask/Casks/with-alt-target.rb")` is significantly more readable for me.